gpcontrib/gg_tables_tracking/isolation2/sql/teardown.sql

```diff
@@ -0,0 +1,4 @@
+--Teardown
+!\retcode ORIGINAL_SPL=$(cat ../../../gpcontrib/gg_tables_tracking/isolation2/results/shared_libs); [ -z "$ORIGINAL_SPL" ] && gpconfig -r shared_preload_libraries || gpconfig -c shared_preload_libraries -v "$ORIGINAL_SPL" ;
```

Comment:
Why so complicated? Why not do it the way I suggested and the way it's already been done in many other similar places?